packages/__docs__/src/Document/index.tsx

```diff
@@ -122,10 +122,12 @@ class Document extends Component<
   renderDescription(doc: SingleChildrenType, description: string) {
     const { id, title } = doc
 
+    const filteredDescription = description.replace('@tsProps', '')
```

Comment:
This was a "fix" for docs, because the "child" components had @tsProps written out in their descriptions.